Frameworks/UIKit.Xaml/ContentDialog.xaml

```diff
@@ -119,7 +119,7 @@
         <ListView.ItemTemplate>
             <DataTemplate x:DataType="local:Item">
                 <!-- Consider making these buttons for accessibility purposes -->
-                <TextBlock Text="{x:Bind Title}" HorizontalAlignment="Left" TextTrimming="WordEllipsis" />
```

Comment:
I'm not aware of any issues, and I didn't debug into the actual failure.  However, this change fixes the issue.  Perhaps we should send a repro to the Xaml team in case this is a bug in the platform.

---
In reply to: [108000838](https://github.com/Microsoft/WinObjC/pull/2321#discussion_r108000838) [](ancestors = 108000838)